components/vf-componenet-rollup/scripts.js

```diff
@@ -20,7 +20,8 @@ vfBannerElixir();
 
 import { vfGaIndicateLoaded } from "vf-analytics-google/vf-analytics-google";
 let vfGaTrackOptions = {
-  vfGaTrackPageLoad: true
+  vfGaTrackPageLoad: true,
+  vfGa4MeasurementId: "G-N5ZLH2KNJP"
```

Comment:
Is this intentional or left by mistake.

Reply:
This is used by the visual-framework.dev doc site, but it probably raises more chances for confusion than help. So I made a clear generic that needs to be updated.